functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_level1group_test.py

```diff
@@ -308,44 +308,4 @@ def test003_level1_and_VMS(self):
             self.lg('-expected error raised %s' % e.message)
             self.assertEqual(e.message,'403 Forbidden')
 
-        """
-    def test004_level1_and_usermangment(self):
```

Comment:
why did u remove it?

Reply:
update && reset user password api is removed

Reply:
/cloudapi/accounts/delete api is removed